extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/conf/LDAPGuacamoleProperties.java

```diff
@@ -181,8 +183,13 @@ private LDAPGuacamoleProperties() {}
      * Property that controls whether or not the LDAP connection follows
      * (dereferences) aliases as it searches the tree.
      */
-    public static final DereferenceAliasesProperty LDAP_DEREFERENCE_ALIASES =
-            new DereferenceAliasesProperty() {
+    public static final EnumGuacamoleProperty<AliasDerefMode> LDAP_DEREFERENCE_ALIASES =
+            new EnumGuacamoleProperty<AliasDerefMode>(
+                "never",     AliasDerefMode.NEVER_DEREF_ALIASES,
+                "searching", AliasDerefMode.DEREF_IN_SEARCHING,
+                "finding",   AliasDerefMode.DEREF_FINDING_BASE_OBJ,
+                "always",    AliasDerefMode.DEREF_ALWAYS
```

Comment:
Yep, because `AliasDerefMode` is part of an API outside of this extension and Guacamole. It's not within our power to add `@PropertyValue` annotations there.